generic_config_updater/field_operation_validators.py

```diff
@@ -126,3 +126,71 @@ def _get_fields_in_patch():
                 return False
 
     return True
+
+
+def read_statedb_entry(table, field):
+    state_db = SonicV2Connector(host="127.0.0.1")
+    state_db.connect(state_db.STATE_DB)
+    return state_db.get(state_db.STATE_DB, table, field)
+
+
+def port_config_update_validator(patch_element):
+
+    def _validate_key(key, port, value):
+        if key == "fec":
+            supported_fecs_str = read_statedb_entry('{}|{}'.format("PORT_TABLE", port), "supported_fecs")
+            if supported_fecs_str:
+                if supported_fecs_str != 'N/A':
+                    supported_fecs_list = [element.strip() for element in supported_fecs_str.split(',')]
+                else:
+                    supported_fecs_list = []
+            else:
+                supported_fecs_list = [ 'rs', 'fc', 'none']
```

Comment:
>supported_fecs_list [](http://example.com/codeflow?start=16&length=19)

Duplicate list of constant. To prevent future mistake when we add one more element, let's define it in a centralized place and reuse.

Reply:
Thanks, I defined the default supported_fecs_list as a constant

Reply:
I see the the same use case in scripts/portconfig. Can we have centralized constant, and reuse in this repo?